app/lib/configuration.js

```diff
@@ -20,6 +20,7 @@ class Configuration {
     this.theme = ''
     this.hotkey = ''
     this.debug = false
+    this.height = 700
   }
 
   load () {
```

Comment:
As in, with this compiled, you are seeing a difference in the height of the Zazu window? I tried it myself and did not notice anything different... :/